airflow/security/permissions.py

```diff
@@ -59,3 +59,15 @@
 ACTION_CAN_ACCESS_MENU = "menu_access"
 DEPRECATED_ACTION_CAN_DAG_READ = "can_dag_read"
 DEPRECATED_ACTION_CAN_DAG_EDIT = "can_dag_edit"
+
+DAG_PERMS = {ACTION_CAN_READ, ACTION_CAN_EDIT}
+
+
+def prefixed_dag_id(dag_id):
```

Comment:
Can we rename this to `permission_for_dag` or something better? WDYT

Reply:
I went back and forth on renaming it since it is being moved from `www/security`, but I'm also happy with a more descriptive name. Related, should we follow up later with deprecating the method in `www/security`?

Reply:
Yup agreed on deprecated method in `www/security`